src/installer.js

```diff
@@ -100,11 +100,31 @@ const getDefaults = function (data) {
           'Utility'
         ],
 
+        pre: undefined,
+        post: undefined,
+        preun: undefined,
+        postun: undefined,
+
         mimeType: []
       }
     })
 }
 
+/**
+ * Read scripts from provided filename and add them to the options
+ */
+const generateScripts = function (options) {
+  const scriptNames = ['pre', 'post', 'preun', 'postun']
+
+  return Promise.all(_.map(options.scripts, (item, key) => {
+    if (_.includes(scriptNames, key)) {
```

Comment:
We should add a TODO here to replace this with `Array.prototype.includes` when we get to Node 6 in this module.